src/main/java/io/confluent/kafka/connect/datagen/DatagenConnector.java

```diff
@@ -37,11 +43,33 @@
   private static Logger log = LoggerFactory.getLogger(DatagenConnector.class);
   private DatagenConnectorConfig config;
   private Map<String, String> props;
+  private Generator generator;
```

Comment:
I don't think we need to use a `Generator` here; instead we can parse schemas directly using the [Avro Java library](https://avro.apache.org/docs/1.8.1/api/java/org/apache/avro/Schema.Parser.html).